apps/payments/next/providers/AppLocalizationProvider.tsx

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use client';
+
+import React, { createContext, useContext, useState, useEffect } from 'react';
+import { ReactLocalization } from '@fluent/react';
+import { useParams } from 'next/navigation';
+import { getLocaleFromRequest } from '@fxa/shared/l10n';
+import {
+  LocalizerClient,
+  LocalizerBindingsClient,
+} from '@fxa/shared/l10n/client';
+
+const AppLocalizationContext = createContext<ReactLocalization | undefined>(
+  undefined
+);
+
+export function AppLocalizationProvider({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const [l10n, setL10n] = useState<ReactLocalization>();
+  const [isLocalized, setIsLocalized] = useState<boolean>(true);
+
+  const params = useParams();
+  const locale = getLocaleFromRequest(params, null);
+
+  useEffect(() => {
+    const setLocalization = async () => {
+      const bindings = new LocalizerBindingsClient();
+      bindings.fetchResource(
+        `${bindings.opts.translations.basePath}/${locale}/main.ftl`
+      );
```

Comment:
Can we sync on Monday? Agreed. I didn't think it would be necessary but then I could not do L37 or L38 so would love for a quick huddle to discuss. 🙂

Reply:
I wasn't able to find a way to call the `LocalizerClient setupReactLocalization` method without passing in bindings. Am I possibly using the wrong method? `setupReactLocalization` does indeed seem to do exactly what I want and for the client as per https://github.com/projectfluent/fluent.js/wiki/ReactLocalization, https://github.com/projectfluent/fluent.js/wiki/React-Tutorial, and what's in `payments-server` [packages/fxa-react/lib/AppLocalizationProvider.tsx](https://github.com/mozilla/fxa/blob/main/packages/fxa-react/lib/AppLocalizationProvider.tsx).

Please let me know.

Reply:
Nope, you are using the right method.

Comment was specifically for lines 34 to 36. If you remove those lines, everything should work as expected.
```
bindings.fetchResource(
  `${bindings.opts.translations.basePath}/${locale}/main.ftl`
);
```